### PR TITLE
Added docs for the Options call

### PR DESCRIPTION
**Pull request by yuce**
## Overview

Adds docs for the `Options` call. Some of the line endings seems to have trailing spaces; those were removed (unintentionally).

Fixes #

## Pull request checklist

- [ ] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [ ] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [ ] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [ ] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [ ] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

File: docs/query-language.md [hunk 1]
```diff
@@ -645,3 +647,41 @@ Sum(field="diskusage")
...
+
+**Spec:**
+
+```
+Options(<CALL>, columnAttrs=<BOOL>, excludeColumns=<BOOL>, excludeRowAttrs=<BOOL>, shards=[UINT ...])
+```
+
+**Description:**
+
+Modifies the given query as follows:
```
Comment (travisturner) [inline docs/query-language.md]: I think we should change these to be something like:
```
argument: action the argument affects
```
So, for example:
* `columnAttrs`: Include column attributes in the result (Default: `false`).
* `excludeColumns`: Include column IDs in the result (Default: `false`).
* `excludeRowAttrs`: Include row attributes in the result (Default: `false`).
* `shards`: Run the query on only the given shards. By default, the query is executed on all shards.

File: docs/query-language.md [hunk 2]
```diff
@@ -645,3 +647,41 @@ Sum(field="diskusage")
...
+* `excludeRowAttrs`: Row attributes are not included in the result (Default: `false`).
+* `shards`: Runs the query only for the given shards. By default, the query is executed for all shards.
+
+**Result Type:** Result type of the given query.
+
+**Examples:**
+
+Return column attributes:
+```request
+Query(Row(f1=10), columnAttrs=true)
```
Comment (travisturner) [inline docs/query-language.md]: `Options`?

File: docs/query-language.md [hunk 3]
```diff
@@ -645,3 +647,41 @@ Sum(field="diskusage")
...
+```
+```response
+{"attrs":{},"columns":[100]}],"columnAttrs":[{"id":100,"attrs":{"foo":"bar"}}
+```
+
+Run the query against shards 0 and 2 only:
+```request
+Query(Row(f1=10), shards=[0, 2])
```
Comment (travisturner) [inline docs/query-language.md]: `Options`?

File: docs/query-language.md [hunk 4]
```diff
@@ -645,3 +647,41 @@ Sum(field="diskusage")
...
+
+**Description:**
+
+Modifies the given query as follows:
+* `columnAttrs`: The result includes column attributes (Default: `false`).
+* `excludeColumns`: Column IDs are not included in the result (Default: `false`).
+* `excludeRowAttrs`: Row attributes are not included in the result (Default: `false`).
+* `shards`: Runs the query only for the given shards. By default, the query is executed for all shards.
+
+**Result Type:** Result type of the given query.
```
Comment (travisturner) [inline docs/query-language.md]: Same result type as `<CALL>`.

File: docs/query-language.md (hunk 1 above)
Comment (yuce) [reply docs/query-language.md]: Re: `shards`. Wouldn't it be more correct to say *for all shards* or maybe *against all shards*?

Comment (travisturner) [reply docs/query-language.md]: My order of preference for those words would be:
```
1) on
2) against
3) for
```
I guess you could get really specific and say something like "Run the query using only the data from the given shards. By default, the entire data set (i.e. data from all shards) is used.

Comment (yuce) [reply docs/query-language.md]: ok, updated

**Pull request merged**